docs/source/contributing/developing.adoc

```diff
@@ -316,6 +316,12 @@ The following options are available:
 
 Tags::
 Use `tags` to ensure that scenarios and features containing at least one of the selected tags are executed.
+To select particular feature you can use its name as a tag, for example: `cmd-config.feature` contains and can be called via `cmd-config` tag.
+There are also tags covering specific areas and usages of {project}:
+
+- `command` tag indicates features which focus on testing of particular subcommands, for example `cmd-addons.feature` 
```

Comment:
"The `command` tag". We need "the" as a definitive here. This applies to all three of these tag descriptions. :)